renovate.json

```
@@ -377,6 +375,26 @@
       "enabled": true,
       "minimumReleaseAge": "7 days"
     },
+    {
+      "groupName": "@elastic/kibana-visualizations test dependencies",
+      "matchDepNames": [
+        "@types/faker",
+        "faker"
+      ],
+      "reviewers": [
+        "team:kibana-visualizations"
+      ],
+      "matchBaseBranches": [
+        "main"
+      ],
+      "labels": [
+        "Team:Visualizations",
+        "release_note:skip",
+        "backport:all-open"
```

```suggestion
        "backport:prev-minor"
```
I believe it's better to just backport to the prev minor, and avoid all-open here